ray-operator/Makefile

```diff
@@ -91,7 +91,7 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
-	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
+	cd config/manager && $(KUSTOMIZE) edit set image kuberay/operator=${IMG}
```

Comment:
should this go to a separate PR?

Reply:
Looks like this is a minor bug fix and could be fine to keep here.
@brucez-anyscale is it the case that `make deploy` was broken due to the image inconsistency?

Reply:
separate PR will be appreciated. should be a quick stamp and merge